modules/conflict_resolver/php/models/resolveddto.class.inc

```diff
@@ -0,0 +1,78 @@
+<?php declare(strict_types=1);
+/**
+ * PHP version 7
+ *
+ * @category Behavioral
+ * @package  Main
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+namespace LORIS\conflict_resolver\Models;
+
+/**
+ * A ResolvedDTO represents a resolved conflict in the conflict
+ * resolver module. It is doing the mapping between the resolved_conflicts
+ * table columns and the JSON object properties returned to the frontend.
+ *
+ * Additionnaly, it implements the DataInstance interface so it can be used by
+ * a Database Provisioner.
+ *
+ * @category Behavioral
+ * @package  Main
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+class ResolvedDTO implements \LORIS\Data\DataInstance
+{
+
+    /**
+     * Implements \LORIS\Data\DataInstance interface for this row.
+     *
+     * @return array
+     */
+    public function jsonSerialize(): array
+    {
+        return [
+            'ResolvedID'          => $this->resolvedid,
```

Comment:
These properties should be defined in the class so we don't have problems once `allow_missing_properties` is finally set to false in phan.